src/client/src/ui/blotter/epics/blotterHighlightEpic.ts

```diff
@@ -0,0 +1,47 @@
+import { applicationConnected } from 'rt-actions'
+import { fromEventPattern } from 'rxjs'
+import { map, switchMapTo, withLatestFrom, delay } from 'rxjs/operators'
+import { ApplicationEpic } from 'StoreTypes'
+import { BLOTTER_ACTION_TYPES, BlotterActions } from '../actions'
+import { combineEpics, ofType } from 'redux-observable'
+import { Action } from 'redux'
+import { Trade } from 'rt-types'
+
+const { highlightTradeAction, removeHighlightTradeAction } = BlotterActions
+const TRADE_HIGHLIGHT_TIME_IN_MS = 3000
+
+const switchHighlight = (trade: Trade) => ({
+  ...trade,
+  highlight: !trade.highlight,
+})
+
+const highlightTradeEpic: ApplicationEpic = (action$, state$, { platform }) => {
+  const interopObservable$ = fromEventPattern(
+    (handler: any) => platform.interop!.subscribe('*', 'highlight-blotter', handler),
+    (handler: any) => platform.interop!.unsubscribe('*', 'highlight-blotter', handler),
+  )
+
+  return action$.pipe(
+    applicationConnected,
+    switchMapTo(interopObservable$),
+    withLatestFrom(state$),
+    map(([message, state]) => {
+      const tradeNotification = message[0].tradeNotification
+      const trade = switchHighlight(state.blotterService.trades[tradeNotification.tradeId])
+      return highlightTradeAction({ trades: [trade] })
+    }),
+  )
+}
+
+const removeHighlightTradeEpic: ApplicationEpic = (action$, state$, { platform }) => {
+  return action$.pipe(
+    ofType<Action, any>(BLOTTER_ACTION_TYPES.BLOTTER_SERVICE_HIGHLIGHT_TRADE),
```

Comment:
I think the second type here is `BLOTTER_ACTION_TYPES.BLOTTER_SERVICE_HIGHLIGHT_TRADE`